repose-aggregator/services/service-client/impl/src/test/scala/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImplTestToo.scala

```diff
@@ -0,0 +1,117 @@
+package org.openrepose.services.serviceclient.akka.impl
+
+import javax.servlet.http.{HttpServletRequest, HttpServletResponse}
+import javax.ws.rs.core.{HttpHeaders, MediaType}
+
+import com.typesafe.scalalogging.slf4j.LazyLogging
+import org.apache.http.HttpResponse
+import org.apache.http.client.methods.HttpGet
+import org.apache.http.impl.client.DefaultHttpClient
+import org.apache.http.params.CoreConnectionPNames
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.core.LoggerContext
+import org.apache.logging.log4j.test.appender.ListAppender
+import org.eclipse.jetty.server.handler.AbstractHandler
+import org.eclipse.jetty.server.{ServerConnector, Request, Server}
+import org.junit.runner.RunWith
+import org.mockito.Matchers._
+import org.mockito.AdditionalMatchers.or
+import org.mockito.Mockito.when
+import org.openrepose.services.httpclient.{HttpClientResponse, HttpClientService}
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.mock.MockitoSugar
+import org.scalatest.{BeforeAndAfter, FunSpec, Matchers}
+
+import scala.collection.JavaConversions._
+
+@RunWith(classOf[JUnitRunner])
+class AkkaServiceClientImplTestToo extends FunSpec with BeforeAndAfter with Matchers with MockitoSugar with LazyLogging {
```

Comment:
I'm not a huge fan of creating a second test class like this. how much work would it be to port the old tests?